test/data/expected/in_tail_deprecated_expected.yaml

```diff
@@ -0,0 +1,11 @@
+logs_module:
```

Comment:
I think we would need to support non-V1 configuration conversion - as they are used in the existing environment, even our catch all configs - so I'd like to confirm with @qingling128 on this. 

If we confirm that we need to support non-V1 configuration, for this PR, let's leave a TODO comment for supporting non-V1 configuraiton, and address later in a separate PR so we can proceed on this PR.

I understood that the current ruby converter doesn't work with non-V1 configuration, but I think there are still non V1 configuration supported

Reply:
What is the non-v1 configuration exactly? If you are referring to using deprecated params like format_firstline rather than using the <parser> dir, then I do support it - I still convert the fields correctly. I just made a separate test case for it for clarity. Or maybe you were referring to some other params?

Reply:
Got it - maybe I should ask what to you mean when you say `deprecated` and how are they not converted into a input config. 

For v1 vs non-v1, I meant Fluent::Config::V1Parser used in https://github.com/googleinterns/observability-agents-config-converter/blob/master/lib/config_parser.rb#L58 and https://github.com/fluent/fluentd/blob/master/lib/fluent/config/parser.rb#L24 . A sample code used in fluentd is: https://github.com/fluent/fluentd/blob/cdedacf7f59f62671e9d16f8a7f991e74d43a79f/lib/fluent/config.rb#L56-L62

If `deprecation` hasn't do with this configuration parser versions, you can ignore this comment.

Reply:
Yep fluentd calls these params as deprecated so I did too, but still mapped them. But yes they are unrelated to the parser versions so will ignore this comment :)